lib/Controller/PageController.php

```diff
@@ -717,34 +717,32 @@ public function createUser( $firstname, $lastname, $mail ) {
 		
 		// if user was created successfully, send him a welcome mail
 		if( $request ) {
+
+            $welcome_mail_message = $this->settings->getSetting( 'welcome_mail_message' );
+
+			// check if password reset is active
+			if( $request && $this->settings->getSetting( 'pwd_reset_url_active' ) ) {
+				$custom_reset_link = '';
```

Comment:
this is redundant, since you are assigning a value to this variable in line 733 and not using the variable between here and 733.